contracts/mockup/modules/IStakingModuleBlockMockup.sol

```diff
@@ -515,6 +515,19 @@ interface IStakingModuleBlockMockup {
         address receiver
     ) external;
 
+    /**
```

Comment:
despite this is a mocking contract it doesn't hurt to add a deprecation warning for sovryn